index.html

```diff
@@ -5,8 +5,31 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Sam - Portofolio</title>
+    <script src="https://kit.fontawesome.com/b39c945343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="assets/style.css" />
 </head>
 <body>
-    
+    <header id="header_menu">
+        <a class="welcome_text">
+            Welcome
+        </a>
+        <a href="#" class="toggleNav">
+            <img src="assets/icons/union.png" alt="union icon" />
+        </a>
+    </header>
+    <section id="main_section">
```

Comment:
- [ ] Kindly add the background-image of this section (using CSS) as shown in the Figma design.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/191605379-070dde8a-c4d9-402d-a155-e84887deae38.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/191605412-012e383c-817f-47b8-bbc7-bc9d3fb079ee.png)